icu_benchmarks/data/split_process_data.py

```
@@ -19,9 +20,11 @@
 def preprocess_data(
     data_dir: Path,
     file_names: dict[str] = gin.REQUIRED,
-    preprocessor: Preprocessor = DefaultClassificationPreprocessor,
+    preprocessor: Preprocessor = PolarsClassificationPreprocessor,
     use_static: bool = True,
     vars: dict[str] = gin.REQUIRED,
+    modality_mapping: dict[str] = {},
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)